PKGBUILD

```diff
@@ -31,8 +30,20 @@ depends=(
 )
 
 _dir="rqt-${pkgver}/rqt_gui"
-source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/ros-visualization/rqt/archive/${pkgver}.tar.gz")
-sha256sums=('3f62f90db254e111483b9a7cb9d486d0bf07c863b2a987808dc3636ee7004c22')
+source=(
+	"241.patch"::"https://github.com/ros-visualization/rqt/pull/241.patch"
```

Comment:
Don't you want this to have the same name as what you use in the prepare function?

Reply:
I am sorry, it is 00:41 here and I am too sleepy to concentrate. I will fix this tomorrow morning. Thank for the review.

Reply:
Done, fix pkgrel, format, and typo in prepare function..